src/components/DataViewFilterMultiselect/DataViewFilterMultiselect.styled.tsx

```diff
@@ -0,0 +1,118 @@
+import styled from "styled-components";
+import theme from "@root/theme";
+
+export const ChipWrapper = styled.div`
+	margin-top: 24px;
+	& .MuiChip-root {
+		display: flex;
+	};
+	& .MuiChip-label {
+		flex: 1 1 0%;
+	};
+`;
+
+export const StyledWrapper = styled.div`
+	padding: 22px 6px 14px;
+
+	& > .topBlock {
+		display: flex;
+		height: 400px;
+		width: 600px;
+	}
+
+	& > .topBlock > .options > .searchBar {
```

Comment:
Since we'll be updating our textfield to use the other one, these styles are no longer needed. (The class stays, because we still want the field to be sticky but only that).
```suggestion
	& > .topBlock > .options > .searchBar {
	   position: sticky;
	   top: 0;
	   margin-bottom: 24px;
	   (Everything else can be deleted)
	}
```